src/applications/representative-appoint/containers/ConfirmationPage.jsx

```diff
@@ -1,97 +1,68 @@
 
-export class ConfirmationPage extends React.Component {
-  componentDidMount() {
-    focusElement('h2');
-    scrollToTop('topScrollElement');
-  }
+export default function ConfirmationPage() {
+  const [signedForm, setSignedForm] = useState(false);
+  const [signedFormError, setSignedFormError] = useState(false);
 
-  render() {
-    const { form } = this.props;
-    const {
-      // submission,
-      formId,
-      data,
-    } = form;
+  const handlers = {
+    onClickDownloadForm: e => {
+      e.preventDefault();
+    },
+    onChangeSignedFormCheckbox: () => {
+      setSignedForm(prevState => !prevState);
 
-    const { fullName } = data;
-
-    return (
-      <div>
-        <div className="print-only">
-          <img
-            src="https://www.va.gov/img/design/logo/logo-black-and-white.png"
-            alt="VA logo"
-            width="300"
-          />
-          <h2>Application for Mock Form</h2>
-        </div>
-        <h2 className="vads-u-font-size--h3">
-          Your application has been submitted
-        </h2>
-        <p>We may contact you for more information or documents.</p>
-        <p className="screen-only">Please print this page for your records.</p>
-        <div className="inset">
-          <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
-            Fill out your form to appoint a VA accredited representative or VSO
-            Claim{' '}
-            <span className="vads-u-font-weight--normal">(Form {formId})</span>
-          </h3>
-          {fullName ? (
-            <span>
-              for {fullName.first} {fullName.middle} {fullName.last}
-              {fullName.suffix ? `, ${fullName.suffix}` : null}
-            </span>
-          ) : null}
-          {/* 
-          {isValid(submitDate) ? (
-            <p>
-              <strong>Date submitted</strong>
-              <br />
-              <span>{format(submitDate, 'MMMM d, yyyy')}</span>
-            </p>
-          ) : null} */}
-          <button
-            type="button"
-            className="usa-button screen-only"
-            onClick={window.print}
-          >
-            Print this for your records
-          </button>
-        </div>
-      </div>
-    );
-  }
-}
-
-ConfirmationPage.propTypes = {
-  form: PropTypes.shape({
-    data: PropTypes.shape({
-      fullName: {
-        first: PropTypes.string,
-        middle: PropTypes.string,
-        last: PropTypes.string,
-        suffix: PropTypes.string,
-      },
-    }),
-    formId: PropTypes.string,
-    submission: PropTypes.shape({
-      timestamp: PropTypes.string,
-    }),
-  }),
-  name: PropTypes.string,
-};
-
-function mapStateToProps(state) {
-  return {
-    form: state.form,
+      if (signedFormError) setSignedFormError(false);
+    },
+    onClickContinueButton: () => {
+      if (!signedForm) {
+        setSignedFormError(true);
+      } else {
+        // Go to instructions page
```

Comment:
I know the button is in a placeholder state waiting for the pdf endpoint. `Go to instructions page` doesn't happen on click, the button does nothing when the box is checked. Just wanted to mention the behavior not matching the comment.